src/selector.js

```diff
@@ -603,13 +604,13 @@ Expr = jQuery.expr = {
 		},
 
 		TAG: function( nodeNameSelector ) {
-			var nodeName = nodeNameSelector.replace( runescape, funescape ).toLowerCase();
+			var expectedNodeName = nodeNameSelector.replace( runescape, funescape ).toLowerCase();
```

Comment:
I thought it might save a bit more to name this `nodeNameExpected` but it was the same. One day I will understand gzip.

Reply:
@dmethvin Minification happens before gzip so source variable names are irrelevant.